src/blocks/card.css

```
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
```

There is no need to use flex-box in this case.
and it would be better to group your styles by [this guide]([codeguide.academy/html-css.html#css-order](https://codeguide.academy/html-css.html#css-order)) Fix in all case
```suggestion
.card {
  padding: 32px 16px 16px;
  width: 200px;

  font-family: "Roboto", sans-serif;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
}
```